lib/active_model/serializer/associations.rb

```diff
@@ -92,7 +92,7 @@ def associations(include_tree = DEFAULT_INCLUDE_TREE)
 
         Enumerator.new do |y|
           self.class._reflections.each do |reflection|
-            next unless include_tree.key?(reflection.name)
+            next unless include_tree.try(:key?, reflection.name)
```

Comment:
This fix is not right, because then you can never explicitly include an association with a custom key.